tests/testthat/test-derive_vars_aage.R

```diff
@@ -162,3 +181,18 @@ test_that("derive_var_agegr_ema - works with age_unit missing and multiple units
     keys = "AGE"
   )
 })
+
+test_that("derive_var_age_years - Error is thrown when age_unit is not proper unit ", {
+  input <- data.frame(AGE = c(12, 24, 36, 48))
+  expect_error(
+    derive_var_age_years(input, AGE, age_unit = "month", new_var = AAGE),
+    "`age_unit` must be one of 'years', 'months', 'weeks', 'days', 'hours', 'minutes' or 'seconds' but is 'month'" # nolint
+  )
+})
+
+test_that("An Error is issued if age_unit is missing", {
```

Comment:
Update `test_that("An Error is issued if age_unit is missing"` to `test_that("derive_var_age_years - An Error is issued if age_unit is missing"` to be consistent with rest of file.